app/containers/Communities/constants.js

```diff
@@ -1,4 +1,6 @@
 export const GO_TO_CREATE_COMMUNITY_SCREEN_BUTTON_ID =
   'app_Communities_GO_TO_CREATE_COMMUNITY_SCREEN_BUTTON_ID';
 
+export const SUBCOMMUNITIES_ID_ARRAY = [];
```

Comment:
SUBCOMMUNITY_IDS_ARRAY better because of subcommunityIds name in other places

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1439/commits/343b49ee805ab5a3b40d1d84982bb2d2f30d7c9c